### DISCUSSION TITLE
How to get a false color with nir, red and green bands?

### DISCUSSION THREAD

**Discussion by efad2022** [developmentseed/titiler#741] [Q&A] [answered]
I'm passing to /stac endpoint the following parameters but I get a white tile rendering :
`
  assets: 'red',
  assets: 'green',
  assets: 'nir',
  rescale: '1,255',
  rescale: '1,255',
  rescale: '1,255',
`    
I appreciate any guideline with this. Thanks

**Comment by vincentsarago** [comment]
@efad2022 we can't really help without reproductible code and access to the data 🤷

**Reply by edeo2021** [reply]
hi @vincentsarago , sure.

I'm sending the following to Titiler server:

> http://localhost:8000/stac/tiles/12/1415/2456.png?tileMatrixSetId=WebMercatorQuad&url=https%3A%2F%2Fearth-search.aws.element84.com%2Fv1%2Fcollections%2Fsentinel-2-l2a%2Fitems%2FS2A_21JYJ_20231226_0_L2A&assets=%5B%20%27red%27%2C%20%27green%27%20%5D&asset_as_band=True&unscale=false&resampling=nearest&reproject=nearest&return_mask=true

In the response I get this:
```
{
    "detail": "'[\"red\",\"green\",\"nir\"]' is not valid, should be one of ['aot', 'blue', 'coastal', 'green', 'nir', 'nir08', 'nir09', 'red', 'rededge1', 'rededge2', 'rededge3', 'scl', 'swir16', 'swir22', 'visual', 'wvp', 'aot-jp2', 'blue-jp2', 'coastal-jp2', 'green-jp2', 'nir-jp2', 'nir08-jp2', 'nir09-jp2', 'red-jp2', 'rededge1-jp2', 'rededge2-jp2', 'rededge3-jp2', 'scl-jp2', 'swir16-jp2', 'swir22-jp2', 'visual-jp2', 'wvp-jp2']"
}
```

**Reply by vincentsarago** [reply]
the `assets` query parameters is of form `list` and to provide multiple values you need to pass assets multiple time 

```
assets=red&assets=green&assets=nir
```

**Reply by efad2022** [reply]
Thanks @vincentsarago , I have found https://devseed.com/sat-ml-training/Scaling where assets are sent a bit a different:

`

r = requests.get(
    f"{titiler_endpoint}/stac/tilejson.json",
    params = {
        "url": url_template.format(id=sceneid[0]),
        "assets": "B04,B03,B02",  # Simple RGB combination (True Color)
        "color_formula": "Gamma RGB 3.5 Saturation 1.7 Sigmoidal RGB 15 0.35",  # We use a rio-color formula to make the tiles look nice
        "minzoom": 8,  # By default titiler will use 0
        "maxzoom": 14, # By default titiler will use 24
    }
).json()
print(r)
`

**Reply by vincentsarago** [reply, marked as answer]
@efad2022 yeah this is pretty old and titiler has changed
